github/orgs_teams.go

```diff
@@ -211,6 +211,32 @@ type OrganizationListTeamMembersOptions struct {
 	ListOptions
 }
 
+// ListChildTeams lists child teams for a team.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/teams/#list-child-teams
+func (s *OrganizationsService) ListChildTeams(ctx context.Context, id int, opt *ListOptions) ([]*Team, *Response, error) {
```

Comment:
Given the name of the receiver type (`OrganizationsService`) and method (`ListChildTeams`), it might not be 100% clear what `id int` refers to. It's quite clear in the endpoint URL itself, being `/teams/:id/teams`, which is why `id` is fine there.

What do you think about renaming the parameter to `teamID` to make it more clear?

Reply:
Good idea, will do.